dist/profile/templates/confluence/vhost.conf

```
@@ -3003,6 +3005,8 @@ RewriteRule "^/display/JENKINS/CSRF\+Protection$" "https://jenkins.io/doc/book/m
 RewriteRule "^/display/JENKINS/FreeBSD$" "https://jenkins.io/doc/book/installing/#freebsd" [NE,NC,L,QSA,R=301]
 RewriteRule "^/display/JENKINS/FreeBSD\+4.9$" "https://jenkins.io/doc/book/installing/#freebsd" [NE,NC,L,QSA,R=301]
 RewriteRule "^/display/JENKINS/How\+to\+report\+an\+issue$" "https://www.jenkins.io/participate/report-issue/" [NE,NC,L,QSA,R=301]
+RewriteRule "^/display/JENKINS//How\+to\+fix\+RequireUpperBoundDeps$" "https://www.jenkins.io/doc/developer/plugin-development/updating-parent/#understanding-requireupperbounddeps-failures-and-fixes" [NE,NC,L,QSA,R=301]
```

Need to remove extra '/' here
```suggestion
RewriteRule "^/display/JENKINS/How\+to\+fix\+RequireUpperBoundDeps$" "https://www.jenkins.io/doc/developer/plugin-development/updating-parent/#understanding-requireupperbounddeps-failures-and-fixes" [NE,NC,L,QSA,R=301]
```